api/app/src/main/kotlin/packit/service/Scheduler.kt

```diff
@@ -0,0 +1,32 @@
+package packit.service
+
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+
+@Service
+class Scheduler(
+        private val packetService: PacketService,
+        private val outpackServerClient: OutpackServerClient
+)
+{
+
+    @Scheduled(fixedDelay = 1000)
```

Comment:
I think polling interval is too short, I do not think data will change every second, we could make it 10 seconds.